.rubocop.yml

```diff
@@ -14,7 +14,7 @@ Metrics/LineLength:
 
 Metrics/MethodLength:
   CountComments: false
-  Max: 26
+  Max: 50
```

Comment:
If this issue is specific to this method due to all the logging it is doing and that is expected (which looks like it), we might just want to disable the cop for this one method instead of increasing the threshold for all methods in the project.
We would just need to add `# rubocop:disable Metrics/MethodLength` to the method `def` line. 
Thoughts? @jsmartt @tmiotto